coordinator/pom.xml

```diff
@@ -70,6 +70,24 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.tencent.rss</groupId>
+      <artifactId>shuffle-storage</artifactId>
```

Comment:
why coordinator need depend on shuffle-storage?

Reply:
To use the hdfs related util functions